Docs update with known issue (#23887)
hashicorp/vault: website/content/partials/known-issues/user-lockout-logging.mdx

### User lockout potential double logging

#### Affected versions

- 1.14.5

#### Issue

> An enhancement introducing additional logging was released in 1.14.5 before intended. There is potential that this additional logging does not trigger or generate double logs. 

Reviewer: ```suggestion
A logging enhancement released before we intended in 1.14.5. In some cases, additional logging does not trigger or generate double logs. 
```

Style correction: write in active voice
